tools/createChangelog.py

```diff
@@ -0,0 +1,24 @@
+import git
```

Comment:
can you add a short docstyle comment to describe what this script does?  (does it update the changelog or create it from scratch? when should it be used? etc)

Reply:
Done.